src/locales/ru/translation.json

```
@@ -0,0 +1,91 @@
+  "Showing {{x}} of {{y}} genomes": "Изображает {{x}} из {{y}} геномов",
+  "sampled between {{from}} and {{to}}": "выборка между {{from}} и {{to}}",
+  "and comprising": "и содержащий",
+  "comprising": "содержащий",
+  "Animation in progress.": "Идет анимация.",
+  "Filtered to": "Фильтр по",
+
+
+  "__Download Modal__": "########################################",
+  "click outside this box to return to the app": "нажмите вне поля чтобы вернуться в приложение",
+  "last updated": "последний раз обновлено",
+  "A full list of sequence authors is available via the TSV files below.": "Полный список авторов доступен в TSV файлах ниже.",
+  "Data usage policy": "Правила использования данных",
+  "Please cite the authors who contributed genomic data (where relevant), as well as:": "Пожалуйста включайте авторов, которые предоставили данные о генах (где уместно), так же как и:",
+  "Download data": "Скачать данные",
+  "Data usage part 1": "Данные представленные здесь предназанчены для быстрого распространения анализа важных патогенов. Неопубликованные данные включены с разрешения создателей и не влияет на их право публикации. ",
+  "Data usage part 2": "Пожалуйста свяжитесь с соответсвующими авторами (список доступен в TSV файлах ниже) если вы собираетесь продолжать исследования используя их данные. Выводные данные, такие как филогенез, можно скачать ниже - пожалуйста свяжитесь с авторами, если уместно.",
+
+  "__Entropy Panel__": "########################################",
+  "Diversity": "Разновидность",
+  "entropy": "энтропия",
+  "events": "события",
+  "Codon {{codon}} in protein {{protein}}": "Кодон {{codon}} в белке {{protein}}",
+  "Nucleotide {{nuc}}": "Нуклеотид {{nuc}}",
+  "Nuc positions {{a}} to {{b}}": "Позиции нуклеоидиов {{a}} до {{b}}",
+  "Num mutations": "Кол-во мутаций",
+  "Negative strand": "Отрицательная цепь",
+  "Positive strand": "Положительная цепь",
+  "Click to color tree & map": "Нажмите чтобы заркасить дерево и карту",
+
+
+  "__Footer__": "########################################",
+  "Filter by {{filterTitle}}": "Фильтровть по {{filterTitle}}",
+  "Data updated": "Данные обновлены",
+
+
+  "__Map panel__": "########################################",
+  "Transmissions": "Передачи",
+  "Geography": "География",
+  "reset zoom": "сбросить приближение",
+  "Reset": "Сброс",
+  "Play": "Пуск",
+  "Pause": "Пауза",
+
+
+  "__Tree (Phylogeny) panel__": "########################################",
+  "Phylogeny": "Филогенез",
+  "Reset Layout": "Сбросить Чертеж",
+  "Click on tip to display more info": "Нажмите на подсказку чтобы посмотреть детали",
+  "Click to zoom into clade": "Нажмите чтобы приблизить филогенетическую ветвь",
+  "Branch leading to": "Ветвь ведущая к",
+  "Number of descendants": "Кол-во потомков",
+  "Nucleotide mutations": "Мутации нуклеотидов",
+  "AA mutations": "Мутации Аминокислот",
+  "protein mutations truncated": "сокращенные мутации белков",
```

```suggestion
  "protein mutations truncated": "мутации белков отображены неполностью",
```
I wasn't quite sure what is exactly displayed here. If this just means that an incomplete list of protein mutations is displayed, then my suggestion on wording is above.